.github/workflows/testing.yml

```diff
@@ -0,0 +1,42 @@
+name: Unit Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+      - cron: '00 4 * * *'  # daily at 4AM
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+      fail-fast: false
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install
+      shell: bash -l {0}
+      run: source continuous_integration/scripts/install.sh
```

Comment:
I personally prefer short and hidden names for CI dirs, i.e. `.ci`, but I won't object too much here, if that approach is used in other places.

I think the advantage of the hidden dirs is that it does not mix with the code dirs when listed. But it's up to you.

Reply:
I kind of agree with your point, but this is a pattern that we have established.